tests/update_stdlib.rs

```diff
@@ -0,0 +1,130 @@
+//! Integration tests related to extrinsic call `update_stdlib`.
+
+mod assets;
+mod mock;
+
+use frame_support::{
+    assert_err, assert_ok, dispatch::DispatchErrorWithPostInfo, pallet_prelude::*,
+};
+use mock::*;
+use move_stdlib::move_stdlib_bundle;
+use move_vm_backend::types::MAX_GAS_AMOUNT;
+use move_vm_backend_common::types::ScriptTransaction;
+
+fn mock_move_stdlib() -> Vec<u8> {
+    assets::read_bundle_from_project("testing-move-stdlib", "testing-move-stdlib")
+}
+
+fn mock_substrate_stdlib() -> Vec<u8> {
+    assets::read_bundle_from_project("testing-substrate-stdlib", "testing-substrate-stdlib")
+}
+
+fn verify_module_error_with_msg(
+    res: DispatchResultWithPostInfo,
+    e_msg: &str,
+) -> Result<bool, String> {
+    if let Err(DispatchErrorWithPostInfo {
+        error: DispatchError::Module(moderr),
+        ..
+    }) = res
+    {
+        if let Some(msg) = moderr.message {
+            return Ok(msg == e_msg);
+        }
+    }
+    Err(format!("{res:?} does not match '{e_msg}'"))
+}
+
+macro_rules! script_transaction {
+    ($bytecode:expr, $type_args:expr, $($args:expr),*) => {
+        {
+            let transaction = ScriptTransaction {
+                bytecode: $bytecode,
+                type_args: $type_args,
+                args: vec![$(bcs::to_bytes($args).unwrap()),*],
+            };
+            bcs::to_bytes(&transaction).unwrap()
+        }
+    }
+}
+
+macro_rules! no_type_args {
+    () => {
+        vec![]
+    };
+}
```

Comment:
This is better, but... wouldn't const be a better option 😃

Reply:
Would be almost the same, but I like the lower case letters more than the upper case ones :laughing: